src/ui/components/UploadScreen/UploadScreen.tsx

```diff
@@ -56,29 +56,35 @@ function Actions({ onDiscard, status }: { onDiscard: () => void; status: Status
   const isDeleting = status === "deleting";
   const shouldDisableActions = isSaving || isDeleting;
 
+  useEffect(() => {      
+    document.getElementById("saveButton").focus(); // for easier upload    
```

Comment:
You can use a ref for this

https://reactjs.org/docs/refs-and-the-dom.html#creating-refs

`saveButtonRef.focus()`